tests/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/Setup/PackageContentTests.cs

```diff
@@ -14,27 +15,29 @@ namespace Microsoft.VisualStudio.Setup
     [UsesVerify]
     public sealed class PackageContentTests
     {
+        private const string XmlSignature = "package/services/digital-signature/xml-signature";
```

Comment:
With the use of MicroBuild signing, there are 3 new `digital-signature` files added to the NuGet packages. This one in specific, `xml-signature`, has a generated name at the time of signing. Thus, it cannot be hard-coded ahead of time as part of the test txt files. So, we filter out checking to see if that file is present.